lib/Onyx.js

```diff
@@ -213,6 +212,15 @@ function isCollectionMemberKey(collectionKey, key) {
     return Str.startsWith(key, collectionKey) && key.length > collectionKey.length;
 }
 
+/**
+ * Splits a collection member key into the collection key part and the ID part.
+ * @param {String} key - The collection member key to split.
+ * @returns {[String, String]} A tuple where the first element is the collection part and the second element is the ID part.
+ */
+function splitCollectionMemberKey(key) {
+    return key.split('_');
```

Comment:
Should this include `isCollectionMemberKey(key) && key.split('_')` so that it's a little bit more safe (say someone tries to use an `_` in the name of a key that's not a collection)?